### ISSUE TITLE
[Android] - Crashes when any accessiblity is used

### ISSUE THREAD

**Issue by husseinm** [mybigday/react-native-external-display#211]
I am using RN 0.61

When the most basic view is used with any accessibility service enabled on the Android device, the app crashes, with the following stack trace:

```
2020-12-09 11:32:55.375 15655-15655/com.petrooutlet.supersonic E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.petrooutlet.supersonic, PID: 15655
    java.lang.IllegalArgumentException: parameter must be a descendant of this view
        at android.view.ViewGroup.offsetRectBetweenParentAndChild(ViewGroup.java:5459)
        at android.view.ViewGroup.offsetDescendantRectToMyCoords(ViewGroup.java:5388)
        at android.view.ViewGroup$ViewLocationHolder.init(ViewGroup.java:7885)
        at android.view.ViewGroup$ViewLocationHolder.obtain(ViewGroup.java:7819)
        at android.view.ViewGroup$ChildListForAccessibility.init(ViewGroup.java:7754)
        at android.view.ViewGroup$ChildListForAccessibility.obtain(ViewGroup.java:7722)
        at android.view.ViewGroup.addChildrenForAccessibility(ViewGroup.java:1978)
        at android.view.ViewGroup.addChildrenForAccessibility(ViewGroup.java:1987)
        at android.view.ViewGroup.addChildrenForAccessibility(ViewGroup.java:1987)
        at android.view.ViewGroup.onInitializeAccessibilityNodeInfoInternal(ViewGroup.java:3076)
        at android.view.View.onInitializeAccessibilityNodeInfo(View.java:6609)
        at android.view.View.createAccessibilityNodeInfoInternal(View.java:6568)
        at android.view.View.createAccessibilityNodeInfo(View.java:6553)
        at android.view.accessibility.AccessibilityRecord.setSource(AccessibilityRecord.java:145)
        at android.view.accessibility.AccessibilityRecord.setSource(AccessibilityRecord.java:119)
        at android.view.View.onInitializeAccessibilityEventInternal(View.java:6505)
        at android.view.View.onInitializeAccessibilityEvent(View.java:6493)
        at android.view.View.sendAccessibilityEventUncheckedInternal(View.java:6358)
        at android.view.View.sendAccessibilityEventUnchecked(View.java:6343)
        at android.view.ViewRootImpl$SendWindowContentChangedAccessibilityEvent.run(ViewRootImpl.java:7479)
        at android.os.Handler.handleCallback(Handler.java:755)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:154)
        at android.app.ActivityThread.main(ActivityThread.java:6141)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:912)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:802)
```

The basic version being used for testing:

EXPO SDK 39, Bare Workflow

```javascript
import React from "react"
import { View, StyleSheet, Text } from "react-native"
import ExternalDisplay, {
  useExternalDisplay,
} from "react-native-external-display"

const CustomerFacingDisplay = () => {
  const screens = useExternalDisplay()

  return (
    <ExternalDisplay
      mainScreenStyle={{ flex: 1 }}
      fallbackInMainScreen={true}
      screen={Object.keys(screens)[0]}
    >
      <View style={{ backgroundColor: "brown", flex: 1, flexDirection: "row" }}>
        <Text>Hello!</Text>
      </View>
    </ExternalDisplay>
  )
}

export default CustomerFacingDisplay
```

**Comment by husseinm** [comment]
By the way, this is an awesome project!

Thank you @jhen0409

**Comment by husseinm** [comment]
My research so far leads me to believe this is some sort of issue with not having some kind of listener on the following class:

https://github.com/mybigday/react-native-external-display/blob/master/packages/react-native-external-display/android/src/main/java/com/externaldisplay/RNExternalDisplayView.java

All the other issues on the internet with this error code are usually because of some kind of missing listener on the class.

I don't have enough experience to know what the delegate is, but it's somewhere there in the Native code for Android of this lib.

**Comment by husseinm** [comment]
Some resolutions for this issue in other libs:

With React Native core's Modal Class:  https://github.com/facebook/react-native/issues/7377 - Resolved by https://github.com/facebook/react-native/commit/57c40d9a6f5a901cf108b72ab18f4bae9fc246e2#diff-6453d554bf967fe816a1aa3735c9fe62

Some Normal Android Apps:

https://stackoverflow.com/questions/26752161/illegalargumentexception-parameter-must-be-a-descendant-of-this-view-at-viewgrou/26752246

https://stackoverflow.com/questions/30585561/another-java-lang-illegalargumentexception-parameter-must-be-a-descendant-of-th

https://stackoverflow.com/questions/27543665/appcompat-java-lang-illegalargumentexception-parameter-must-be-a-descendant-of

https://stackoverflow.com/questions/7100555/preventing-catching-illegalargumentexception-parameter-must-be-a-descendant-of

**Comment by ratheeps** [comment]
Have the same issue. When press reload, works perfectly!

**Comment by husseinm** [comment]
Reload does not fix anything for me.

**Comment by jayan2019** [comment]
It worked for me when I run with debugger reload. And when disable all accessibility, working smoothly!

**Comment by husseinm** [comment]
That's the point.

When you run w/ Debugger that is not a fair comparison because you have the browser runtime being utilized, not Hermes or whatever the device is using.

The issue being reported is that when Accessibility is used the app crashes, disabling definitely "resolves" the issue, but I need Accessibility enabled for my MDM solution.

I am also testing on a modified Android 7.1.2 device (Android TV Box w/ two monitors)

**Comment by jhen0409** [comment]
I'm sorry that I have no chance to test the Android real device with secondary display at the moment. Also, I can't reproduce it with Android Emulator with `simulate secondary displays` developer option.

> Some resolutions for this issue in other libs:
> 
> With React Native core's Modal Class: facebook/react-native#7377 - Resolved by facebook/react-native@57c40d9#diff-6453d554bf967fe816a1aa3735c9fe62

Have you tried this solution? It should be possible to add it directly to RNExternalDisplayView.

**Comment by husseinm** [comment]
It appears adding that exact patch to RNExternalDisplayView works!

I will make a PR for you.

**Comment by husseinm** [comment]
[The PR has been made](https://github.com/mybigday/react-native-external-display/pull/216), if you could @jhen0409 , please accept and bump up the version/publish to master.

Thank you!

**Comment by jhen0409** [comment]
Thank you @husseinm!

Fixed in v0.5.6.

**Comment by jayan2019** [comment]
Thank you @husseinm & @jhen0409  for the timely response & the fix! I truly appreciate it!!
